Cloud Application Manager/Managed Services/MSA Networking.md

```diff
@@ -29,9 +29,52 @@ This article is meant to assist users of Managed Services Anywhere (MSA) in the
 
 All Managed Services Anywhere (MSA) customers or those interested in enabling MSA within their Cloud Application Manager Providers.
 
-![Managed Services Anywhere Network Diagram](../../images/cloud-application-manager/MSAnetwork_9-26-19.png)
+
+### MSA Network Diagram
+
+Network connections between customer servers, MSA management appliance, and CenturyLink.
+
+[![Managed Services Anywhere Network Diagram](../../images/cloud-application-manager/MSAnetwork_9-26-19.png)](../../images/cloud-application-manager/MSAnetwork_9-26-19.png)
```

Comment:
Is this an existing image? Otherwise, it should be added to the PR, or updated if required.

Reply:
Yes, the image is existing.  I don't know how it was generated, but if we want to remove port 80 it does need to be updated.  @thomasbroadwell I think that came from you?

Reply:
Hey @thomasbroadwell, could you please provide/update the diagram? Thanks!

Reply:
Hey @pra-ctl, please update link to diagram with: `MSA_Network-3-3-20.png`